dist/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Javascript Capstone</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Italianno&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo">
+          <img src="../src/imgs/logo.jpg" alt="logo"/>
+        </div>
+        <ul>
+          <li>
+            <a href="#">
+              Meals(<span id="items-num">6</span>)
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              Reservation
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              About
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <ul id="list">
+
+    </ul>
+    <section class="comments-section"></section>
+    <footer class="footer">
+      <div class="logo">
+        <img src="../src/imgs/logo.jpg" alt="logo"/>
+      </div>
+      <div class="copyright-names">
+        <small>&#169; 2021 
+        <a href="#">Ahmed. Al-Farouq &</a><br>
+        <a href="#">Francis Ugorji.N.</a><br>
+        Some Rights Reserved</small>
+      </div>
+    </footer>
```

Comment:
- [ optional ] It will be nice to add some more styling to the footer section to give it a better visual.